releases/Process/Stabilization Process.md

```diff
@@ -0,0 +1,85 @@
+-   **Hardening Process:** The process of allowing the stabilization branch to be tested with zero changes over a period of time, giving confidence that the code base is of high enough quality to release. This occurs at the end of the stabilization process.
+
+
+## Exception Process  
+The Exception process is managed in a Github project - the process is currently documented in the description of the project [https://github.com/o3de/o3de/projects?type=classic](https://github.com/o3de/o3de/projects?type=classic) .  
+There is an open discussion regarding streamlining the exception process: [https://github.com/o3de/sig-release/discussions/106](https://github.com/o3de/sig-release/discussions/106)    
+
+
+## Goals of the Stabilization process
+
+1.  **Stabilize the Release:** Ensure we ship the most stable version of O3DE we can.
+2.  **Separate Release preparation from day-to-day development:** We permit bug fixing and stabilization to occur without hindering developers that are working on new features.
+3.  **Stabilize the O3DE project overall:** Our oricess ensures that bug fixes made for stabilization are reflected in the development branch.
+  
+## Stabilization Process:
+
+*Note: during this process, development still continues as normal in the development branch. The development branch is never locked and never requires exceptions.*  
+  
+### Precondition:  
+
+1.  (Release Manager) coordinates with the Technical Steering Committee, the Open 3D Foundation Marketing Committee, and the SIGs to determine a release date.
+
+### Phase 1 - Setup
+
+1.  (Release Deployment Engineer) Currently, 6 weeks before a planned release, a stabilization branch is created. This branch is named Stabilization/<version number>. Example; for the 22.10 release we would create a Stabilization/2210 branch.
+2.  (Release Deployment Engineer) The development branch, in its current state, is integrated to the newly created stabilization branch, preferably on the same day the stabilization branch is created.
+3.  (Release Manager) publishes a note in sig-all indicating that we have begun release stabilization. Encourage the community to test, report bugs, and fix them in the stabilization branch.
+4.  (Release Manager) Coordinates with sigs to create a schedule of stabilization → development integrations and maintainers responsible for those integrations.
+
+### Phase 2 - Stabilization
+
+1.  When fixing bugs for stabilization and/or a release, please fix these bugs in stabilization and let the stabilization integrators handle getting these fixes to development branch. “Cherry picking” of changes between development and stabilization branches should not be done because it can lead to problematic integrations.
```

Comment:
Just curious. How does the stabilization integrator do it without cherry-picking?

Reply:
In this case, cherry picking is integrating specific commits from Development to Stabilization. While integrators can work around a cherry pick (a specific PR pulled in from Development to Stabilization), it can complicate the full integration back from Stabilization to Development, particularly if other changes are submitted on top of the cherry pick. Also Development branch is still moving ahead during stabilization, and likely has changes that will break Stabilization. Integrating from Development to Stabilization can inadvertently bring a breaking change with it. We are not saying "you should never cherry pick from Development" as there are times when it is necessary. The guidance is just to reduce the number of potential issues and conflicts for integrators, and attempt to keep Stabilization 'stable'. This guidance also helps to avoid an integrator needing specific knowledge in order to understand cherry picked PRs. There is too much code to expect anyone to have complete knowledge of all systems.

Reply:
Thanks for the context, Joe!